django_ipam/templates/admin/django-ipam/subnet/change_form.html

```diff
@@ -48,7 +48,7 @@ <h3>{% trans 'Subnet Visual Display' %}</h3>
     subnet_tree.push(
     {% for subnet_queryset in subnet_tree %}
       {% for subnet_item in subnet_queryset %}
-          {subnet_id: "{{ subnet_item.pk }}",
+          {subnet_id: "{{ subnet_item.id }}",
```

Comment:
Instead of changing to `id` here, is it possible to change `id` to `pk` in the backend? 
I think that will be better because users may create their own models from the Abstract model and any model will always have `pk`! 
What do you think? :smile:

Reply:
@atb00ker I have looked at the code and can't seem to find a way to get the `pk` from the backend as you proposed. Do you have any solution for this?

Reply:
If I am not wrong, this [function](https://github.com/openwisp/django-ipam/blob/master/django_ipam/base/admin.py#L32) should help. 
Specifically, note ID is used here:
- https://github.com/openwisp/django-ipam/blob/master/django_ipam/base/admin.py#L44
- https://github.com/openwisp/django-ipam/blob/master/django_ipam/base/admin.py#L54
- https://github.com/openwisp/django-ipam/blob/master/django_ipam/base/admin.py#L63

I think getting `pk` in this function should do the trick! :smile:

Reply:
I forgot to review it in time, but thanks! :smile: